pom.xml

```diff
@@ -118,30 +141,10 @@
 
     <build>
         <plugins>
-            <plugin>
```

Comment:
Is there a reason why you removed the enforcer plugin? Or was this the last version being compatible with Java 7?

Reply:
honestly saying I don't see need to use enforcer plugin here, since pom here is relative simple and don't utilise functional that tied to specific maven version

Reply:
The enforcer plugin was configured back before we started to use the maven wrapper. So it's pretty much useless now since we specify the maven version that we want to use.